zaas-client/src/main/java/org/zowe/apiml/zaasclient/service/internal/ZaasJwtService.java

```diff
@@ -245,8 +242,8 @@ private String extractToken(SimpleHttpResponse response) throws ZaasClientExcept
         String token = "";
         int httpResponseCode = response.getCode();
         if (httpResponseCode == 204) {
-            var vals = response.getHeaders().get(HttpHeaders.SET_COOKIE).getValue().split(";");
-            var apimlAuthCookie = Stream.of(vals).filter(v -> v.startsWith(zassConfigProperties.getTokenPrefix())).map(v -> v.substring(v.indexOf("=") + 1)).findFirst();
+            var cookies = response.getHeaders().get(HttpHeaders.SET_COOKIE).stream().map(header -> header.getValue().split(";")).flatMap(Arrays::stream).toList();
```

Comment:
This is not correct implementation, see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie. Semicolon doesn't means just split cookies.

Reply:
My bad, fixed.